rocky/katalogus/views/katalogus.py

```diff
@@ -73,7 +73,7 @@ def sort_queryset(self, queryset, sort_options):
         if sort_options == "a-z":
             return queryset
         if sort_options == "z-a":
-            return reversed(queryset)
+            return list(reversed(queryset))
```

Comment:
If the `queryset` object is already a subscribable/ indexable object, you can also reverse it with `queryset[::-1]`

Reply:
Ideally, this would just be a sort/order argument that gets passed on to the katalogus api. the same goes for most of the filters applied in this page.

Reply:
Yes indeed, I introduced https://github.com/minvws/nl-kat-coordination/issues/396 a while ago for exactly this problem.